client/src/main/java/com/tigrisdata/db/client/StandardTigrisClient.java

```diff
@@ -71,71 +58,9 @@ public static StandardTigrisClient getInstance(TigrisConfiguration tigrisConfigu
   }
 
   @Override
-  public TigrisDatabase getDatabase(String databaseName) {
+  public TigrisDatabase getDatabase(String projectName) {
```

Comment:
The project name should already be known at this point, so we don't need to ask the project name?

Reply:
fixed.